File: TJ Code pre Fall-2026/ProgrammingTeam/ThePlank/ThePlank.py
numWays = 0
def checkLength(length) -> None:
    global numWays
    if(length == 0):
        numWays += 1
    if(length >= 3):
        checkLength(length - 3)
    if(length >= 2):
        checkLength(length - 2)
    if(length >= 1):
        checkLength(length - 1)
    

numMeter = int(input())
checkLength(numMeter)
print(numWays)

File: TJ Code pre Fall-2026/ProgrammingTeam/ThePlank/ThePlank.java
import java.util.*;
class ThePlank
{
    static int checkLength(int length)
    {
        if(length < 0)
        {
            return 0;
        }
        else if(length == 0)
        {
            return 1;
        }
        else 
        {
            int sum = 0;
            for(int i = 1; i < 4; i++)
            {
                sum += checkLength(length - i);
            }
            return sum;
        }
        
    }
    public static void main(String[] args)
    {
        Scanner sc = new Scanner(System.in);
        int numMeter = sc.nextInt();
        System.out.println(checkLength(numMeter));
        
    }
}